odkx-src/services-using.rst

```diff
@@ -62,6 +62,23 @@ Within this page you can enter a new :guilabel:`username` and :guilabel:`passwor
 
 If you want to log out of your current user without logging into a new user, click the :guilabel:`Log Out` button. This does not require a network connection.
 
+.. _services-using-qr-scanner:
+
+Authenticating Users with QR-scanner
+---------------------------------------
+
+Users can use this option as an alternative to above method by logging in using QR code Scanner, that can be generated by app-designer.
```

Comment:
This sentence was a bit unclear/repetitive to me. Maybe "Users can alternatively log in using the QR code scanner, with QR codes generated by the ODK-X app-designer"